src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-param-reassign */
+const chalk = require('chalk');
+
+const ENVIRONMENTS = require('../../../constants/environments');
+const getApiClient = require('./api');
+const getAllPages = require('./getAllPages.graphql');
+
+const ENABLED_ENVIRONMENTS = new Set([
+  ENVIRONMENTS.LOCALHOST,
+  ENVIRONMENTS.VAGOVDEV,
+]);
+
+function pipeApiDataIntoMetalsmith(contentData, files) {
+  // Generates a Drupal index page at /drupal, which is a useful (temporary) resource
+  // for checking Drupal payloads.
+
+  const {
+    data: {
+      nodeQuery: { entities: pages },
+    },
+  } = contentData;
+
+  let drupalIndexPage =
+    '<h1>The following pages were provided by Drupal:</h1><ol>\n';
+
+  for (const page of pages) {
+    const {
+      entityUrl: { path },
+    } = page;
+
+    const jsonPath = `drupal${path}.json`;
+
+    drupalIndexPage += `<li><a href="/${jsonPath}">${path}</a></li>`;
+
+    files[jsonPath] = {
+      ...page,
+      contents: Buffer.from(JSON.stringify(page, null, 4)),
+    };
+  }
+
+  drupalIndexPage += '</ol>';
+
+  files['drupal/index.html'] = {
+    contents: Buffer.from(drupalIndexPage),
+  };
+
+  // eslint-disable-next-line no-console
+  console.log(chalk.rgb(73, 167, 222)('Drupal index page written to /drupal.'));
```

Comment:
This console statement is to make the Drupal work more transparent to other devs, but maybe it makes more sense not to have that.

Reply:
Yeah that's a great idea. I'd love to see that.